libcudacxx/include/cuda/std/detail/libcxx/include/algorithm

```diff
@@ -2892,12 +2892,12 @@ private:
     static const _Working_result_type _Rp = _Engine::_Max - _Engine::_Min
                                           + _Working_result_type(1);
 #else
-    static _LIBCUDACXX_CONSTEXPR const _Working_result_type _Rp = _Engine::max() - _Engine::min()
+    static constexpr const _Working_result_type _Rp = _Engine::max() - _Engine::min()
```

Comment:
Do we want to simplify this to `constexpr` instead of `constexpr const`

Reply:
Probably a good idea.